### PR TITLE
Frontend: exchanges change to use or filter for payment methods

### PR DESCRIPTION
**Pull request by thisconnect**

### CODE REVIEW

**Comment by x1ddos** [comment]
@thisconnect if you rebase onto latest master, the wright-pad situation should be gone.

**Comment by thisconnect** [comment]
PTAL

File: frontends/web/src/routes/exchanges/exchanges.tsx [hunk 1]
```diff
@@ -62,23 +62,19 @@ class Exchanges extends Component<Props, State> {
...
         { t }: RenderableProps<Props>,
         {
-        methods,
+        method,
         region,
     }) {
         const results = this.data
             .filter(({ regions }) => !region || regions.includes(region))
-            .filter(({ payment }) => !methods.length || methods.some(m => payment.includes(m)))
+            .filter(({ payment }) => !method || payment.includes(method))
```
Comment (x1ddos) [inline frontends/web/src/routes/exchanges/exchanges.tsx:77]: shouldn't this be `... || payment === method` now?

Comment (thisconnect) [reply frontends/web/src/routes/exchanges/exchanges.tsx:77]: `region` and `method` are string to filter from, but 
`regions` and `payment` are arrays, see https://github.com/digitalbitbox/bitbox-wallet-app/blob/f42efa2f6962de963af8e4b7d180aa3dfe391300/frontends/web/src/routes/exchanges/exchanges-data.ts#L8-L9

Comment (thisconnect) [reply frontends/web/src/routes/exchanges/exchanges.tsx:77]: Should I rename `payment ` to be `payments` ?

**Comment by thisconnect** [comment]
rebased on top of upstream/master

File: frontends/web/src/routes/exchanges/exchanges.tsx (hunk 1 above)
Comment (x1ddos) [reply frontends/web/src/routes/exchanges/exchanges.tsx:77]: Oh, my bad. I think it's not worth renaming. :+1:

**Pull request merged**